### PR TITLE
Restaurants update

### PR DESCRIPTION
**Pull request by Jackimaru96**
- Added in more restaurants and users
- Updated README to include how to import data in tables using csv files
- Updated restaurants view to include cuisine
- Updated restaurant searchbox to standardise with homepage

DONT MERGE YET;
Changing code resulted in searchbox in homepage not being able to work. 
Wailun will be resolving the conflict! Thanks @wailunlim

### CODE REVIEW

File: views/restaurants.pug [hunk 1]
```diff
@@ -52,12 +59,19 @@ block content
...
                         th Restaurant Names
+                        th Cuisine
 
                 tbody
                     each restaurant in restaurants
                         tr
                             td
                                 a(href="/restaurants/" + restaurant.rname)= restaurant.rname
+                            td
+                                <cuisinetext>
```
Comment (Jackimaru96) [inline views/restaurants.pug:70]: Ignore `cuisinetext`, will remove next time the class doesn't work dk why so had to use td.even to change the hover for the cuisine

**Comment by wailu** [comment]
Remove your .idea files first!

File: controllers/restaurants.controller.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ const db = require('../db/index');
 exports.showRestaurants = async (req, res, next) => {
   try {
     const restaurants = await queryDbFromReqQuery(
-      'SELECT DISTINCT rname FROM OwnedRestaurants NATURAL JOIN HasTimeslots',
+      'SELECT rname, cuisine FROM OwnedRestaurants ORDER BY rname',
```
Comment (wailu) [inline controllers/restaurants.controller.js:7]: Actually this change will break the search; but I'll fix it in the next PR so shouldn't be a problem.

File: views/.idea/checkstyle-idea.xml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (wailu) [inline views/.idea/checkstyle-idea.xml]: To remove

**Review by wailu — CHANGES_REQUESTED** [review]
.idea files to be removed first before we merge.

**Review by wailu — APPROVED** [review]

**Pull request merged**